src/content/guides/how-to-revoke-token-access/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to revoke smart contract access to your crypto funds
+description: A how to guide on revoking exploitative smart contract token access
+lang: en
+---
+
+# How to: Revoke smart contract access to your crypto funds
+
+With this guide you will learn how to view a list of all smart contracts you have allowed access to your funds and also how to cancel them. 
+
+Sometimes malicious developers build backdoors into smart contracts that will allow access to the funds of unaware users who interact with their projects. What often happens is that such platforms ask the user for permission to spend an **unlimited number of tokens**.
```

Comment:
```suggestion
Sometimes hackers build backdoors into smart contracts to gain access to the funds of unaware users who interact with the smart contract. What often happens is that such platforms ask the user for permission to spend an **unlimited number of tokens**.
```

Reply:
Thoughts on 'hackers' vs 'malicious developers'? It's not strictly accurate, but it's a simpler mental model that might make more sense generally IMO.

Reply:
not sure i agree here, on a personal level i imagine hacker to be a person exploiting some thirdparty software like hacking a bridge while malicious developer is someone who writes the software with the purpose of attracting and exploiting users.